ci/azure/macos.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     maxParallel: 11
     matrix:
       py35_np_110:
-        ENV_FILE: ci/azure-macos-35.yml
```

Comment:
is the name of this file crucial for azure? (e.g. the .yml vs .yaml)?

Reply:
I honestly have no idea.  The fact that this name can be wrong without breaking the CI is troubling.

Reply:
ok, can you rename these as well and see?

Reply:
I must have miscommunicated.  The answer to the previous question is "apparently not".  In the status quo, the azure/windows-py27.yml and azure/windows.yml files have `ENV_FILE: ci/azure-windows-27.yml` that point to non-existent files, and azure doesn't appear to notice.

Reply:
ahh interesting. so any idea how this was working? or is it just picking up the wrong env?